packages/preferences-controller/src/PreferencesController.test.ts

```diff
@@ -423,10 +423,10 @@ describe('PreferencesController', () => {
     expect(controller.state.showIncomingTransactions['0x1']).toBe(false);
   });
 
-  it('should set smartTransactionsOptInStatus', () => {
+  it('should set smartTransactionsOptInStatus to false', () => {
     const controller = setupPreferencesController();
-    controller.setSmartTransactionsOptInStatus(true);
-    expect(controller.state.smartTransactionsOptInStatus).toBe(true);
+    controller.setSmartTransactionsOptInStatus(false);
+    expect(controller.state.smartTransactionsOptInStatus).toBe(false);
```

Comment:
Does this test really need to be only for the false case though? Maybe it should be `should set smartTransactionsOptInStatus`

and test both setting to  `false` (`false` first since default is now `true`) then `true`.